src/Language/Visitor/EnterReturnValue.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Digia\GraphQL\Language\Visitor;
+
+final class EnterReturnValue extends AbstractReturnValue
```

Comment:
Couldn't we just get by with one class, e.g. `VisitorResult`? If not, then I'd call these two classes `VisitorEnterResult` and `VisitorLeaveResult`.

Reply:
I suppose we could, it's just that enter has one more possible value than leave

Reply:
Ok fair enough, let's keep it like this then. 👍